scripts/publishNPMPackages.js

```diff
@@ -24,7 +24,7 @@ import { invariant } from '@kiwicom/js';
       '@kiwicom/babel-preset-kiwicom',
       '@kiwicom/eslint-config',
       '@kiwicom/eslint-config-nitro',
-      '@kiwicom/fetch',
+      '@adeira/fetch',
```

Comment:
Could you please remove everything else from here (and keep only this line)? It won't be 1:1 since not everything is necessary to publish now.